tools/testrunner.py

```diff
@@ -28,6 +28,47 @@
 from common_py.system.platform import Platform
 
 
+# Defines the folder that will contain the coverity info.
```

Comment:
Using `coverity` here is confusing to me. Why don't you use `coverage`?

Reply:
Thanks your notice. Fixed.